docs/sdkx-react-native/support-tools.mdx

```diff
@@ -0,0 +1,154 @@
+---
+sidebar_position: 20
+title: Helpshift APIs
+description: "Helpshift provides a way to use each functional element of the SDK separately based on your requirements."
+---
+
+import {
+  Admonition,
+  CodeBlock,
+  Tabs,
+  TabItem,
+  LatestSdkVersion,
+  Centered,
+  Image,
+  Intro,
+  SideBySide,
+  DownloadButton,
+  Steps,
+  Step,
+} from "@site/src/components/forDocs";
+
+# Helpshift APIs {#support-tools}
+
+<Intro>
+
+Helpshift provides a way to use each functional element of the SDK separately based on your requirements.
+You can call only specific screens from the SDK based on the functionality you want to expose in your app.
+
+</Intro>
+
+<Admonition type="info" title="Note">
+
+All the public APIs in the SDK should be called after initializing the SDK via [Helpshift.install() API](/sdkx-react-native/getting-started#start-using)
+
+</Admonition>
+
+## Integrating Contact Us & In App Messaging {#conversation-view}
+
+<Centered width="major">
+
+![newConversation.png](/static/books/sdkx_android/show_conversation.png "newConversation.png")
+
+</Centered>
+
+You can use this API call to allow a user to directly send feedback or start a new conversation without having to first view FAQs.
+
+This API call will show the conversation screen. Once a conversation is started, it will continue until it is resolved or rejected by the agent on Helpshift Dashboard.
+
+To configure SDK, you can pass various config options as a parameter. Check out more configuration options [here].(/sdkx-react-native/sdk-configuration)
+
+Example :
+
+```
+//Create your config here.
+const config = {};  
+
+// pass the config in the api
+showConversation(config);
+
+```
+
+## Integrating FAQs {#faqs-view}
+
+<Centered >
+
+![showFAQs.png](/static/books/sdkx_android/show_faqs.png "showFAQs.png")
+
+</Centered>
+
+You can use this API to provide a way for the user to invoke the purpose-built help/support section in your app. This is the easiest approach to enable help in your app as it bundles all the capabilities of the Helpshift SDK in a simple and intuitive interface. You can wire this API call to a **"Help"** or **"Support"** action in your app.
+
+To configure the FAQs screen, you can pass various config options as a parameter. Check out more config options [here](/sdkx-react-native/sdk-configuration)
+
+Example :
+
+```
+showFAQsWithConfig({
```

Comment:
showFAQsWithConfig({
  faq: <FAQ_SECTION_ID> 
}); 
Show faq is with config, why are giving faqId here?

Reply:
updated.